src/components/LoadingMessage/LoadingMessage.less

```diff
@@ -0,0 +1,33 @@
+.lucid-LoadingMessage {
+
+	.lucid-Icon {
+		display: inline-block;
+	}
+
+	@LoadingMessage-padding: @size-standard;
```

Comment:
Hm, if that's the convention, i can stick to it, but i'm not sure why we wouldn't using real scopes when they're available.